### PR TITLE
Show svc weight as 100 if app blank

### PR DESCRIPTION
**Pull request by cbron**
Issue: https://github.com/rancher/rio/issues/739

Weight calc on ps was summing on app string, but app string can be blank and it was counting those as the same app.

### CODE REVIEW

File: cli/cmd/ps/ps_services.go [hunk 1]
```diff
@@ -121,13 +121,17 @@ func (p *Ps) services(ctx *clicontext.CLIContext) error {
 
 		weight := 0
 		if service.(*riov1.Service).Status.ComputedWeight != nil && *service.(*riov1.Service).Status.ComputedWeight > 0 {
-			totalWeight := 0
-			for _, subService := range services {
-				if service.(*riov1.Service).Spec.App == subService.(*riov1.Service).Spec.App && subService.(*riov1.Service).Status.ComputedWeight != nil {
-					totalWeight += *subService.(*riov1.Service).Status.ComputedWeight
+			if service.(*riov1.Service).Spec.App == "" {
```
Comment (StrongMonkey) [inline cli/cmd/ps/ps_services.go]: @cbron You should always use `appAndVersion` method to get app.

Comment (cbron) [reply cli/cmd/ps/ps_services.go]: Fixed

File: cli/cmd/ps/ps_services.go [hunk 2]
```diff
@@ -114,20 +115,26 @@ func (p *Ps) services(ctx *clicontext.CLIContext) error {
...
 			continue
 		}
 
 		weight := 0
 		if service.(*riov1.Service).Status.ComputedWeight != nil && *service.(*riov1.Service).Status.ComputedWeight > 0 {
-			totalWeight := 0
-			for _, subService := range services {
-				if service.(*riov1.Service).Spec.App == subService.(*riov1.Service).Spec.App && subService.(*riov1.Service).Status.ComputedWeight != nil {
-					totalWeight += *subService.(*riov1.Service).Status.ComputedWeight
+			app, _ := services.AppAndVersion(service.(*riov1.Service))
```
Comment (StrongMonkey) [inline cli/cmd/ps/ps_services.go]: Hmm, app will not be empty in this case.

Comment (cbron) [reply cli/cmd/ps/ps_services.go]: Ohh, I bet its just falling down to next check and that check works correctly now. I'll remove.

**Pull request merged**